Libplanet.Net/Protocols/KademliaProtocol.cs

```diff
@@ -150,22 +150,10 @@ public class KademliaProtocol : IProtocol
             }
             catch (DifferentAppProtocolVersionException e)
             {
-                AppProtocolVersion expected = e.ExpectedVersion, actual = e.ActualVersion;
                 _logger.Debug(
-                    $"Different version encountered during {nameof(AddPeersAsync)}().\n" +
-                    "Expected version: {ExpectedVersion} ({ExpectedVersionExtra}) " +
-                    "[{ExpectedSignature}; {ExpectedSigner}]\n" +
-                    "Actual version: {ActualVersion} ({ActualVersionExtra}) [{ActualSignature};" +
-                    "{ActualSigner}]",
-                    expected.Version,
-                    expected.Extra,
-                    ByteUtil.Hex(expected.Signature),
-                    expected.Signer.ToString(),
-                    actual.Version,
-                    actual.Extra,
-                    ByteUtil.Hex(actual.Signature),
-                    actual.Signer
-                );
+                    e,
+                    "Different version encountered during {FName}().",
+                    nameof(AddPeersAsync));
```

Comment:
Don't we need detailed log of version information? In some environment `e` is not logged (ex. testing)

Reply:
I haven't really looked into this, but is there a reason why `e` can't be logged? If possible, I think the right solution would be to print `e` instead of nearly duplicating `e.Message`. 🤨